### PR TITLE
[CODE STYLE] Reducing use of else statements, adding early returns [com_checkin]

### PR DESCRIPTION
**Pull request by ghost**
This series of Pull Requests improve some basic code styles.
Mostly if/else syntax improvements, focussing on removing else statements which follow returns (or continues in loops).
Also de-nesting of if structures by adding early returns/elses.

Example:

```
function foobar($foo, $bar)
{
   if($foo > $bar)
   {
      return $foo;
   } else {
      return $bar;
   }
}
```

Can be changed to:

```
function foobar($foo, $bar)
{
   if($foo > $bar)
   {
      return $foo;
   }

   return $bar;
}
```

PS: This only illustrates the 'issue' with the useless else statements.

Current PRs in this series:
- #6286 [com_admin] **MERGED**
- #6299 [com_banners]
- #6300 [com_cache]
- #6301 [com_categories]
- #6305 [com_checkin]
- #6282 [com_finder]
- #6283 [lib.tags]

### CODE REVIEW

File: administrator/components/com_checkin/controller.php [hunk 1]
```diff
@@ -45,21 +45,22 @@ public function checkin()
...
 			JError::raiseWarning(500, JText::_('JLIB_HTML_PLEASE_MAKE_A_SELECTION_FROM_THE_LIST'));
-		}
-		else
-		{
-			// Get the model.
-			$model = $this->getModel();
 
-			// Checked in the items.
-			$this->setMessage(JText::plural('COM_CHECKIN_N_ITEMS_CHECKED_IN', $model->checkin($ids)));
+			$this->setRedirect($redirect);
```
Comment (wilsonge) [inline administrator/components/com_checkin/controller.php]: does there not need to be a return here?

Comment (nonumber) [inline administrator/components/com_checkin/controller.php]: Good catch! :)

**Comment by yvesh** [comment]
Haven't had any problems during testing and overlooking the code - you could do even some more cleanup though ;-)<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/6305">issues.joomla.org/joomla-cms/6305</a>.</sub>

**Comment by nonumber** [comment]
Yes, this is just a first run. Only tackling certain types of code. Small steps. Small steps. :)

**Comment by nonumber** [comment]
If this doesn't find any interest within a week or 2, I'll close this PR and create new PRs with smaller fixes (one fix per PR).

**Comment by Duke3D** [comment]
@nonumber Patch seems to be redundant to 6301 with regard to changes to com_categories controller.  Unable to apply this patch while patch 6301 is active.  Should these patches be merged with each other for testing?  Reverting 6301 and applying 6305 seems to work but not extensively tested.  Please advise.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/6305">issues.joomla.org/joomla-cms/6305</a>.</sub>

**Comment by nonumber** [comment]
Thanks @Duke3D. Removed the com_categories changes. Should indeed not have been in this PR.

**Comment by nonumber** [comment]
Due to lack of interest (read: nobody testing), this is now closed.

**Pull request closed without merging**